src/nodes.coffee

```diff
@@ -1948,6 +1971,13 @@ exports.Arr = class Arr extends Base
       obj = obj.unwrapAll()
       obj.eachName iterator
 
+  astType: -> 'ArrayExpression'
```

Comment:
Once we've added AST support for `Assign`, this will have additional logic based on whether it's an "assigned-to" (ie left-hand-side) array or not, since those are `type: 'ArrayPattern'`

Similar goes for `Splat`, which as a left-hand-side will be `type: 'RestElement'`